flutter_web_auth_2/example/lib/main.dart

```diff
@@ -95,7 +101,7 @@ class MyAppState extends State<MyApp> {
 
       // Windows needs some callback URL on localhost
       req.response.write(
-        (Platform.isWindows || Platform.isLinux)
+        (Platform.isLinux)
```

Comment:
`desktop_webview_window` claims to also work on Linux. Do you have a specific reason why you don't also enable it on that platform?

Reply:
My original intention was to fix #73 because I had multiple users complaining about it. But theoretically it should be possible to use the same approach for Linux too. I haven't tested it though.

Reply:
The linux version of that library requires `webkit2gtk-4.1` as a dependency installed on the system, which might not be available on "older" systems (https://packages.ubuntu.com/search?keywords=webkit2gtk). How would you like to proceed?

Reply:
Interesting, so there are different kinds of requirements on the webview approach.
I think, the best way to handle this is to let the developer decide which approach to take.
Maybe, it is possible to provide a configuration option within `FlutterWebAuth2Options` which has an enum (or something?) representing the approach that should be chosen.
Depending on that, `flutter_web_auth_2` chooses either your approach or the current one.
Do you think, that might be possible?

Reply:
I think something like that should be possible. I'll try to implement something like that. However the compiling computer will still require that dependency to be installed for linking/compiling.

Reply:
Thank you very much! If you are stuck somehow, just let me know. Maybe I can help!